packages/@lwc/template-compiler/src/index.ts

```diff
@@ -29,6 +29,9 @@ export function parse(source: string, config: Config = {}): TemplateParseResult
     return parseTemplate(source, state);
 }
 
+// Export as a named export as well for easier importing in certain environments (e.g. Jest)
+export { compile };
+
```

Comment:
I needed this to unbreak our Jest tests. Jest is weird about default exports.

Reply:
I'm wondering if this will break anything else in the DX ecosystem, if they import `@lwc/template-compiler` directly. I'm fine with this change, but we should keep an eye out for issues that might arise because of this change.

Reply:
I'm hoping we'll be fine, and that this is just an internal issue with TypeScript+Jest on our source code, but it's impossible to know for sure.

FWIW the current package [uses](https://unpkg.com/browse/@lwc/template-compiler@2.45.2/dist/commonjs/index.js) `exports.default = compile` in the `dist` CJS file, and the new `dist` file does the same thing.